masterlist.yaml

```diff
@@ -13180,7 +13187,27 @@ plugins:
     req: [ 'Better descriptions SSE.esp' ]
   - name: 'Better Descriptions SSE - YOT Patch.esp'
     req: [ 'YOT - Your Own Thoughts.esp' ]
-
+  - name: 'RealisticWaterTwo - Unique Region Names.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: [ 'RealisticWaterTwo - Unique Region Names - Cutting Room Floor.esp' ]
+        condition: 'active("RealisticWaterTwo - Unique Region Names - Cutting Room Floor.esp")'
+      - <<: *alreadyInX
+        subs: [ 'RealisticWaterTwo - Unique Region Names - MissingEZs_AllExteriors.esp' ]
+        condition: 'active("RealisticWaterTwo - Unique Region Names - MissingEZs_AllExteriors.esp")'
+  - name: 'Unique Region Names.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/56452/' ]
+    group: *cellSettingsGroup
```

Comment:
It shouldn't be put this late, it wipes outs changes from too many other mods.
The previous version I made myself was stripped down to avoid that exact problem.

```suggestion
```

Reply:
Yes, I wasn't very conservative here: some tests I took in my load orders made it look good enough, but I trust your judgement.